src/auto/cell_area.rs

```diff
@@ -112,9 +121,17 @@ pub trait CellAreaExt {
     fn set_focus_cell<T: IsA<CellRenderer>>(&self, renderer: &T);
 
     fn stop_editing(&self, canceled: bool);
+
+    fn connect_add_editable<F: Fn(&Self, &CellRenderer, &CellEditable, &cairo::RectangleInt, TreePath) + 'static>(&self, f: F) -> u64;
+
+    fn connect_apply_attributes<F: Fn(&Self, &TreeModel, &TreeIter, bool, bool) + 'static>(&self, f: F) -> u64;
+
+    fn connect_focus_changed<F: Fn(&Self, &CellRenderer, TreePath) + 'static>(&self, f: F) -> u64;
+
+    fn connect_remove_editable<F: Fn(&Self, &CellRenderer, &CellEditable) + 'static>(&self, f: F) -> u64;
 }
 
-impl<O: IsA<CellArea>> CellAreaExt for O {
+impl<O: IsA<CellArea> + IsA<Object>> CellAreaExt for O {
```

Comment:
Currently - not, so we adding it to all traits with signal.
You seen this in #394